### ISSUE TITLE
Small thumbnail in target event of edge

### ISSUE THREAD

**Issue by josefspr** [daat-hamakom/maps#118]
When there is a reference/path/communication, I think the event that is connected to the selected event should be lighted up or perhaps the 2 sides of edge should be treated as a group. What do you think ?

**Comment by mushon** [comment]
yes, treated as a group. I think this is already implemented? or is it just a group that contains a path?
![image](https://cloud.githubusercontent.com/assets/27123/15701121/05023af0-27e1-11e6-9806-06d3c35e9f4d.png)

**Comment by josefspr** [comment]
The events are lighted up because they are in group. The 2 edges (or at least the one that is not open) should light up event if not in a group.

**Comment by josefspr** [comment]
target event of edge will open a small thumbnail.
